config/secrets.defaults.yml

```diff
@@ -2,6 +2,14 @@
 test:
   ibm_cloud_power_defaults: &ibm_cloud_power_defaults
     api_key: IBM_CLOUD_POWERVS_API_KEY
+    cloud_instance_id: IBM_CLOUD_POWERVS_INSTANCE_ID
+    ibmcloud_region: ibm_cloud_powervs_region
+    vm_name: rdr-miq-test
+    key_pair_name: miq-pvs
+    public_network_name: public-192_168_165_88-29-VLAN_2039
+    power_network_name: miq-net
+    image_name: '7100-05-05'
+    sys_type: s922
```

Comment:
Do you actually want to hide these?  `config/secrets*` is for replacing "real" values with ones which are publicly safe to commit to a vcr_cassette.  This looks like you should have these in a different config file or just straight up in the terraform state

Reply:
I have moved these setting into the new `*config/powervs-testsettings.yml*` file